test/test/helpers/use-mockery/README.md

```diff
@@ -0,0 +1,42 @@
+# Use Mockery
+
+##### Load and unload mockery in a safe manner
+
+Lots of tests use `mockery` to mock out the underlying requirements of a module under test. Loading and using `mockery` can be tricky with a lot of tests. You have to remember to unload it after use, use a clean cache, etc. This module takes care of automating those tasks for you.
+
+Generally, we see `mockery` based mocking as a last resort. We prefer passing dependencies when we can, as `mockery` can be slow and invasive.
+
+## Usage
+
+```js
+import useMockery from 'test/helpers/use-mockery';
+
+// THINGS REQUIRED HERE WILL NOT HAVE MOCKED DEPENDENCIES
+
+describe('my test suite', () = {
+
+	before( () => {
+		// THINGS REQUIRED HERE WILL NOT HAVE MOCKED DEPENDENCIES
+	} );
+
+	useMockery();
+
+	before( () => {
+		// you can require things that have mocked dependencies now!
```

Comment:
When should I be doing the mocking? Should we include example `mockery.registerMock` or `mockery.registerSubstitute` ?